activation/handler_v2.go

```diff
@@ -808,7 +808,20 @@ func (h *HandlerV2) checkDoubleMerge(ctx context.Context, tx sql.Transaction, at
 		zap.Stringer("smesher_id", atx.SmesherID),
 	)
 
-	var proof wire.Proof
+	var blob sql.Blob
+	version, err := atxs.LoadBlob(ctx, tx, other.Bytes(), &blob)
```

Comment:
btw this blob reads in handler are very expensive, i didn't raise it yet but to make it viable to run code will have to get rid of them on happy path (e.g when there is no equivocation)